taxi/forms.py

```diff
@@ -0,0 +1,34 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Car, Driver
+
+
+class LicenseNumberFormMixin(forms.Form):
+    license_number = forms.CharField()
```

Comment:
The `license_number` field lacks validation to ensure it consists of 8 characters, with the first 3 being uppercase letters and the last 5 being digits. Consider adding a `RegexValidator` to enforce this format.